e2e/cypress/integration/accessibility/accessibility_dropdowns_spec.js

```diff
@@ -89,81 +84,69 @@ describe('Verify Accessibility Support in Dropdown Menus', () => {
-        cy.get('#headerInfo button').should('have.attr', 'aria-label', 'main menu').and('have.class', 'a11y--active a11y--focused').click();
+        // # Open team menu
+        cy.uiGetLHSHeader().click();
 
         // * Verify the accessibility support in the Main Menu Dropdown
-        cy.get('#sidebarDropdownMenu').should('have.attr', 'aria-label', 'main menu').and('have.class', 'a11y__popup').and('have.attr', 'role', 'menu');
+        cy.findByRole('menu').
+            should('exist').
+            and('have.attr', 'aria-label', 'main menu').
+            and('have.class', 'a11y__popup');
 
         // * Verify the first option is not selected by default
-        cy.get('#sidebarDropdownMenu .MenuItem').children().eq(0).should('not.have.class', 'a11y--active a11y--focused');
-
-        // # Press tab
-        cy.focused().tab();
+        cy.uiGetLHSTeamMenu().find('.MenuItem').
+            children().eq(0).
+            should('not.have.class', 'a11y--active a11y--focused').
+            focus();
 
         // * Verify the accessibility support in the Main Menu Dropdown items
         const menuItems = [
-            {id: 'accountSettings', label: 'Account Settings dialog'},
             {id: 'invitePeople', label: 'Invite People dialog'},
             {id: 'teamSettings', label: 'Team Settings dialog'},
             {id: 'manageMembers', label: 'Manage Members dialog'},
-            {id: 'createTeam', text: 'Create a Team'},
             {id: 'joinTeam', text: 'Join Another Team'},
             {id: 'leaveTeam', label: 'Leave Team dialog'},
-            {id: 'integrations', text: 'Integrations'},
-            {id: 'marketplaceModal', label: 'Marketplace dialog'},
-            {id: 'systemConsole', text: 'System Console'},
-            {id: 'helpLink', text: 'Help', directLink: true},
-            {id: 'gettingStarted', text: 'Getting Started'},
-            {id: 'keyboardShortcuts', text: 'Keyboard Shortcuts'},
-            {id: 'reportLink', text: 'Report a Problem', directLink: true},
-            {id: 'nativeAppLink', text: 'Download Apps', directLink: true},
-            {id: 'about', label: `About ${siteName} dialog`},
-            {id: 'logout', text: 'Log Out'},
+            {id: 'createTeam', text: 'Create a Team'},
         ];
 
         menuItems.forEach((item) => {
-            if (!item.shouldSkip) {
-                // * Verify that the menu item is focused
-                cy.get('#sidebarDropdownMenu').find(`#${item.id}`).should('be.visible').within(() => {
-                    if (item.label) {
-                        cy.findByLabelText(item.label).should('have.class', 'a11y--active a11y--focused');
-                    } else if (item.directLink) {
-                        cy.findByText(item.text).should('have.class', 'a11y--active a11y--focused');
-                    } else {
-                        cy.findByText(item.text).parent().should('have.class', 'a11y--active a11y--focused');
```

Comment:
We'll no longer go on specific check since those are more on implementation details.